legato-api/src/main/java/org/opendaylight/legato/LegatoServiceController.java

```diff
@@ -0,0 +1,384 @@
+
+			if (response.get().isSuccessful()) {
+				LOG.info("call Success = {}, response = {} ", response.get()
+						.isSuccessful(), response.get().getResult());
+				LOG.info("evcId = {}, UUID = {} ", evcId, response.get()
+						.getResult().getService().getUuid().getValue());
+
+				EVC_UUIDMap.put(evcId, response.get().getResult().getService()
+						.getUuid().getValue());
+
+				LOG.info("======== {} ", EVC_UUIDMap.toString());
+
+				Optional<Evc> OptionalEvc = LegatoUtils.readEvc(
+						dataBroker,
+						LogicalDatastoreType.CONFIGURATION,
+						InstanceIdentifier
+								.create(MefServices.class)
+								.child(CarrierEthernet.class)
+								.child(SubscriberServices.class)
+								.child(Evc.class,
+										new EvcKey(new EvcIdType(evcId))));
+
+				// Add Node in OPERATIONAL DB
+				if (OptionalEvc.isPresent()) {
+					UpdateEvcOnOperationalDB(OptionalEvc.get());
+				}
+
+			} else
+				LOG.info("call Failure = {} >> {} ", response.get()
+						.isSuccessful(), response.get().getErrors());
+		} catch (Exception ex) {
+			LOG.error("Error in callCreateConnectionService(). Err: ", ex);
+		}
+	}
+
+	private void callUpdateConnectionService(
+			UpdateConnectivityServiceInput updateConnectivityServiceInput,
+			String evcId) {
+		try {
+			Future<RpcResult<UpdateConnectivityServiceOutput>> response = this.prestoConnectivityService
+					.updateConnectivityService(updateConnectivityServiceInput);
+
+			if (response.get().isSuccessful()) {
+				LOG.info("call Success = {}, response = {} ", response.get()
+						.isSuccessful(), response.get().getResult());
+
+				Optional<Evc> OptionalEvc = LegatoUtils.readEvc(
+						dataBroker,
+						LogicalDatastoreType.CONFIGURATION,
+						InstanceIdentifier
+								.create(MefServices.class)
+								.child(CarrierEthernet.class)
+								.child(SubscriberServices.class)
+								.child(Evc.class,
+										new EvcKey(new EvcIdType(evcId))));
+
+				// update EVC Node in OPERATIONAL DB
+				if (OptionalEvc.isPresent()) {					
+					LegatoUtils.deleteFromOperationalDB(InstanceIdentifier
+							.create(MefServices.class)
+							.child(CarrierEthernet.class)
+							.child(SubscriberServices.class)
+							.child(Evc.class, new EvcKey(new EvcIdType(evcId))), dataBroker);
+
+					UpdateEvcOnOperationalDB(OptionalEvc.get());
+				}
+
+			} else
+				LOG.info("call Failure = {} >> {} ", response.get()
+						.isSuccessful(), response.get().getErrors());
+		} catch (Exception ex) {
+
+			LOG.error("Error in UpdateConnectivityServiceInput(). Err: ", ex);
+		}
+	}
+
+	private boolean callDeleteConnectionService(
+			DeleteConnectivityServiceInput deleteConnectivityServiceInput) {
+		try {
+			this.prestoConnectivityService
+					.deleteConnectivityService(deleteConnectivityServiceInput);
+			return true;
+
+		} catch (Exception ex) {
+			LOG.error("Fail to call callDeleteConnectionService(). Err: ", ex);
+			return false;
+		}
+	}
+	
+	private void UpdateEvcOnOperationalDB(Evc evc) {
```

Comment:
please use Java naming convention for method names, should be ``updateEvcOnOperationalDb``
Please make sure you fix it in the whole pull request